src/helpers/types.ts

```diff
@@ -48,7 +48,7 @@ export const sectionColorsDataMap = {
     secondary: null,
   },
   researchLibrarySchwarzman: {
-    primary: "section.research-library.schwartzman",
+    primary: "section.research-library.schwarzman",
```

Comment:
There's no "t" in "Schwarzman" so updated it throughout the repo. I noticed this when trying to apply the design token color. @bigfishdesign13

Reply:
Good catch.  The token name is fine in the VDL.

Reply:
Do we need to leave the typo spelling in there along with the correct spelling to avoid a breaking change?

Reply:
No, this is a change that might break but is easy enough and should be corrected.